server/models/group.js

```diff
@@ -21,6 +21,11 @@ const GroupSchema = new mongoose.Schema({
     type: EscalationPolicy.schema,
     default: EscalationPolicy.defaultEscalationPolicy()
   },
+  requests: [{
```

Comment:
Can you make this something a bit more verbose? Something like `userRequests` or `joinRequests`? Request could mean way too many things.